### PR TITLE
refactor: Cleaner file structure

### PR DESCRIPTION
**Pull request by spydon**
* This moves away from the `src` directory style, since that is to be used in packages, not in apps (since there are no dependents of an app).
* Since the root was a mix of barrel files and classes the barrel files moved in to their respective directories.
* The vote class was named `user.dart` in the ratings package, renamed it to `vote.dart`.

### CODE REVIEW

File: packages/app_center/l10n.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-arb-dir: lib/src/l10n
+arb-dir: lib/l10n
```
Comment (spydon) [inline packages/app_center/l10n.yaml:1]: @d-loose will this need a setup change in weblate?

Comment (d-loose) [reply packages/app_center/l10n.yaml:1]: Yes :melting_face:

Comment (spydon) [reply packages/app_center/l10n.yaml]: Moved it back in [ac7fcb5](https://github.com/ubuntu/app-center/pull/1698/commits/ac7fcb5120b3e118e1d54648fc27f8ea2434720c)

**Review by d-loose — APPROVED** [review]
Thanks, LGTM :+1:

**Pull request merged**